IFC4x3/Properties/c/Comment_1TwvmDDDf2JPk_$_XfLkbr/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@j-weil: Can you provide an example on a comment (which is not Description)?

Reply:
maybe something that comes form a review, an opinion or additonal info relation to a project? We commonly use comments to highlight something "commen iC: borehole log inconsistent", ...
However I think in IFC or BIM prpject in general there are mutiple alternative ways to treat this, with bcf etc.
so it might not be important here

Reply:
Precisely @j-weil, if you want to exchange issues, it might be a better way to just exchange the issues instead of a complete model. As you say, in those cases BCF is probably much more prudent.

Reply:
Ok, I will remove!